README.md

```diff
@@ -340,14 +340,28 @@ The `$status` variable contains the status code of the command, and the
 `$output` variable contains the combined contents of the command's standard
 output and standard error streams.
 
+If invoked with one of the following as the first argument, `run`
+will perform an implicit check on the exit status of the invoked command:
+
+    =N  expect exit status N (0-255), fail if otherwise
+    '!' expect nonzero exit status (1-255), fail if command succeeds
```

Comment:
I think `run !` without the `''` is perfectly fine. My bash documentation says this:

> Reserved words are words that have a special meaning to the shell.  The following words are recognized as **reserved when unquoted** and either the **first word of a simple command** (see SHELL GRAMMAR below) or the **third word of  a case or for** command: 
> **!** case  coproc  do done elif else esac fi for function if in select then until while { } time [[ ]]

And something like the following just works:
```
$ echo ! word
! word
```

Reply:
Oh, nice, thank you. It looks a little uncomfortable to my eye, due to my fear that bash will interpret it as a history event... but I see there's no conflict, and I can learn.